public/data/SiddhiBhanushali.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "Siddhi Bhanushali",
-  "bio": "Web development & open source enthusiast",
+  "bio": "Web developer & open source enthusiast",
```

Comment:
I don't think we should update other people's profile - if that is what they put, then they should change it if they wish to do so

Reply:
> I don't think we should update other people's profile - if that is what they put, then they should change it if they wish to do so

Good idea 😅 closing this PR. Thank you, Eddie.